app-admin/syslog-ng/syslog-ng-3.7.3.ebuild

```diff
@@ -1,138 +0,0 @@
-# Copyright 1999-2017 Gentoo Foundation
-# Distributed under the terms of the GNU General Public License v2
-
-EAPI=5
-PYTHON_COMPAT=( python2_7 )
-inherit autotools python-single-r1 eutils multilib systemd versionator
-
-MY_PV=${PV/_/}
-MY_PV_MM=$(get_version_component_range 1-2)
-DESCRIPTION="syslog replacement with advanced filtering features"
-HOMEPAGE="http://www.balabit.com/network-security/syslog-ng"
-SRC_URI="https://github.com/balabit/syslog-ng/releases/download/${P}/${P}.tar.gz"
-
-LICENSE="GPL-2+ LGPL-2.1+"
-SLOT="0"
-KEYWORDS="alpha amd64 arm arm64 hppa ia64 ~mips ppc ppc64 ~s390 ~sh sparc x86 ~x86-fbsd"
```

Comment:
This dropped the only stable version for alpha, arm, arm64, hppa, ia64, ppc, ppc64 and sparc; and only ~arch version for mips, s390, sh and x86-fbsd (from the listed ~arch ones only ~mips matters though). Not OK.

Reply:
@leio So what's the actual way to fix this? Revert the drop for now, add all arches you mentioned as ~ARCH beside ~arm, open a bug for ~arm and later on stabilise everything again?
(Cc'ing @hydrapolic)

Reply:
It's already reverted and syslog-ng with ivykis is being keyworded.